### ISSUE TITLE
Issues with Build Directory

### ISSUE THREAD

**Issue by J-M-Smith-Aero** [Tudat/tudat#580]
Hi there

The build settings showed the build directory in red text, and I could not build the libraries. Previously, when configuring the project, the output in the file below was given.  Do the errors seen here mean that the build is not possible?

Kind regards

Jake

[Generating Tudat Bundle in Qt.txt](https://github.com/Tudat/tudat/files/3829067/Generating.Tudat.Bundle.in.Qt.txt)

**Comment by DominicDirkx** [comment]
Hi Jake,

You say "I could not build the libraries." Could you please provide your compilation output? The output you post seems to contain your CMake output (given in Qt under General Messages) only,

Best,

Dominic

**Comment by J-M-Smith-Aero** [comment]
Hi Dominic

Thank you for the quick reply. I don't seem to be able to get that information now, as I restarted the program and it now does not need to need to compile; but I am able to build the project, which I am doing right now. My only concern was that the build configuration says "Debug", whereas in the screenshot on the tutorial it says "Default". This does not seem to be an option for me, however.

I will inform on here if there are any more issues. Below I have also attached the command prompt script as I was concerned by the first few lines. However, after that it did not seem to have an issue. I have stored the code in the directly /C: directory on my computer, so hopefully there is no problem.

Kind regards

Jake

[Tudat Cloning Command.txt](https://github.com/Tudat/tudat/files/3830534/Tudat.Cloning.Command.txt)

**Comment by DominicDirkx** [comment]
Hi Jake,

Instead of 'Debug', put it to 'Release', if there is no option for 'Default', and then compile.
The first line in your terminal output is, indeed, nothing to worry about,

Best,

Dominic

**Comment by J-M-Smith-Aero** [comment]
Thanks Dominic. I installed successfully, and none of the unit tests failed.
However, I have a small issue when it comes to running an application. I see that the build settings in my programme and the screenshot provided differ slightly; though I have only ticked the relevant ones. When I attempted to configure this I got a message saying that it had failed because I do not have Doxygen installed. Surely this should not be an issue, as I have CMake?
